.github/workflows/ci.yml

```diff
@@ -121,14 +123,20 @@ jobs:
             -DANDROID_PLATFORM=25 \
             -DCMAKE_TOOLCHAIN_FILE=$ANDROID_NDK_HOME/build/cmake/android.toolchain.cmake \
             -DALSOFT_EMBED_HRTF_DATA=TRUE \
-            -DALSOFT_REQUIRE_OPENSL=ON",
+            -DALSOFT_REQUIRE_OPENSL=ON \
+            -DALSOFT_REQUIRE_OBOE=ON \
+            -DOBOE_SOURCE=oboe",
             build_type: "Release"
           }
     steps:
     - uses: actions/checkout@v4
       with:
         fetch-depth: '0'
 
+    - name: Setup Oboe for Android
+      if: ${{ contains(matrix.config.name, 'Android') }}
+      run: git clone https://github.com/google/oboe
```

Comment:
Would probably be good to add `--depth=1` to not pull the whole history. Maybe also have it grab the latest release if possible, instead of the head.